src/utils/common-utils/common-utils.spec.js

```diff
@@ -18,4 +21,31 @@ describe('common-utils', () => {
       expect(result).toEqual(input);
     });
   });
+
+  it('should set the summary error to have focus and scroll into view', () => {
```

Comment:
Please can you put this in a `describe('focusErrors', () => {…})` block?

Reply:
Actioned